### DISCUSSION TITLE
Welcome to snapcast Discussions!

### DISCUSSION THREAD

**Discussion by badaix** [snapcast/snapcast#738] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gwm** [comment]
Hi... I use a couple of mopidy instances and a DLNA renderer to serve up my local music collection, podcasts, radio streams and casts from my phone via AirAudio. This is all curated by the wonderful snapcast server listening to 3 fifos (one for each of the mopidy instances and one for the dlna renderer). This works fantastically well for whole house audio.

problem: snapweb is confusing

I just upgraded to 0.22 in order to try and get snapserver integrated with mopidy-iris. This brought websockets which it turns out is required for the iris integration (not json rpc). It works, occasionally. But the clients are unrecognisably named and it keeps disconnecting. It looks like this integration is not yet mature/working.

But 0.22 also comes with built in web client so I connect to that from a browser (http://server:1780/). But I can't see my named streams ("stream1", "stream2" and "air"), clients are named as expected but my streams are probably related to "groups" in the web client. Groups are just showing as "Group 1" "Group 2" and "Group 3". I can move known clients between unknown groups. On the web page these client are always listed in a group called "default:playing"

Am I misunderstanding how this works? 

btw. snapdroid works perfectly and I can move my clients between streams, adjust volumes etc. The problem is that I am limited to an android client to adjust these - browser based would be more flexible (laptop, ipad etc).
